contracts/forwarder/Forwarder.sol

```diff
@@ -60,12 +60,16 @@ contract Forwarder is IForwarder {
 
         bytes memory callData = abi.encodePacked(req.data, req.from);
         require( gasleft()*63/64 >= req.gas, "FWD: insufficient gas" );
+        // solhint-disable-next-line not-rely-on-time
+        require(req.validUntil==0 || req.validUntil > block.timestamp, "FWD: request expired");
```

Comment:
Inconsistent spacing for boolean expressions. Unfortunately, solhint does not seem opinionated about that.